content/doc/book/managing/system-configuration.adoc

```diff
@@ -13,3 +13,114 @@ endif::[]
 
 = Configuring the System
 
+Jenkins can be configured at many levels and with many different techniques.
+Your final configuration depends directly on which problems you want to solve.
+
+Jenkins has a specific area all configurations items can be accessed.
```

Comment:
The list below is not complete, maybe it makes sense to add ", etc." at the end or so